src/assets/sass/protocol/components/_card.scss

```diff
@@ -165,96 +165,3 @@
         max-width: 300px;
     }
 }
-
-/* -------------------------------------------------------------------------- */
-// A small card with an icon and no link.
-
-.mzp-c-card-picto {
```

Comment:
I'm moving this card class to it's own scss file, as we have enough variations on cards now we should probably try to avoid building up a monolith file.